qa/L0_java_memory_growth/Simple.java

```diff
@@ -834,7 +841,7 @@ output0, output1, new FloatPointer(input0_data), new FloatPointer(input1_data),
 
       Runnable runnable =
         () -> { 
-          if(ValidateMemoryGrowth(.05f)){
+          if(ValidateMemoryGrowth(.25f)){
```

Comment:
How about abstracting all these factors in a constraint object?

Reply:
Thanks for the suggestion. Do you mean a class to hold all the constants that gets passed into this function? I can also make these command line arguments (with default values) and just pass them into this function, if that works.

Reply:
Yes. The cli arguments works great! I was little confused with the Simple.java nomenclature.

Can we rename the Simple.java to memory_growth_test.java so that we are consistent with all other tests?

Reply:
Sure thing. Thanks for clarifying!

Reply:
25% is a large tolerance. Although I understand that the base memory usage is small so 25% may end up be a small amount, we should keep the small percentage tolerance and use the fixed minimum delta added above as additional checking

Reply:
I realized the print message needs to be fixed (decimal points), but the last run had a 15.2% memory "growth" due to the memory snapshots starting out around 17MB and later going to 15MB.

If we keep a 5% tolerance threshold, tests will sometimes fail. 25% seems like a better spot to mitigate intermittence, since the memory would grow way beyond that in 12 hours if there's a memory leak (as we've seen in the past). Or what threshold would you prefer? I can also make it so that the threshold is higher for the long-running test only.

Reply:
I was thinking that we will fail the test if the validation doesn't satisfy both % and delta constraints

Reply:
We can do that, but if we leave it at 5%, we should expect future failures. This PR fixes the error due to it allowing a higher percentage of variation (25%) to account for the non-deterministic garbage collection in Java.

Also, note that the new constraint I added is the maximum memory (total), but I can change it to the absolute delta if preferred.